README.md

```diff
@@ -72,87 +68,70 @@ export default () => {
+In addition to these component-specific properties you will also have all the properties of the Text Input component that can be found on [its site](https://developer.dxc.com/halstack/6/components/text-input/#props).
 
-#### HalAutocomplete Component Example
+#### HalAutocomplete Example
 
 ```JSX
-import React, { useState, useEffect } from "react";
-import {HalAutocomplete} from "@dxc-technology/halstack-react-hal";
+import React, { useState } from "react";
+import { HalAutocomplete } from "@dxc-technology/halstack-react-hal";
 
 export default () => {
-  const [autocompleteValue, changeAutocompleteValue] = useState("")
-  const onChange = (newValue) => {
+  const [autocompleteValue, changeAutocompleteValue] = useState("");
+  const onChange = ({ newValue }) => {
     changeAutocompleteValue(newValue);
   };
 
   return (
-    <div>
-
-      <HalAutocomplete
-        url="https://..."
-        propertyName="full-name"
-        label="Full Name"
-        onChange={onChange}
-        value={autocompleteValue}
-      ></HalAutocomplete>
-    </div>
+    <HalAutocomplete
+      url="https://..."
+      propertyName="full-name"
+      label="Full Name"
+      onChange={onChange}
+      value={autocompleteValue}
+    />
   );
 };
-
 ```
 
 ### useHalResource Hook
 
-#### useHalResource Usage
-
-```JSX
-import { useHalResource } from "@dxc-technology/halstack-react-hal";
-```
-
 #### useHalResource Parameters
 
 | Name                                    | Default | Description                                                                                                                                                                                                                                                                                     |
 | :-------------------------------------- | :------ | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | url: `string`                           |         | The URL of the resource. `Required`                                                                                                                                                                                                                                                             |
 | headers: `Object`                       |         | Contains the http headers to be sent along with the http requests to the url indicated in the `url` prop. `Optional`                                                                                                                                                                            |
```

Comment:
As `url` is an abbreviation, it should be written in uppercase `URL`.

Reply:
Done.